youtube_dl/extractor/stolaf.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import js_to_json
+
+
+class StOlafIE(InfoExtractor):
+    _VALID_URL = r'^https?://(?:www\.)?stolaf\.edu/multimedia/play/\?e=(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://www.stolaf.edu/multimedia/play/?e=573',
+        'info_dict': {
+            'id': '573',
+            'ext': 'mp4',
+            'title': 'Senior Soloists Concert',
+            'description': 'St. Olaf Orchestra & Senior Soloists',
+            'thumbnail': 'http://www.stolaf.edu/multimedia/components/poster/e573',
+        },
+        'params': {
+            'skip_download': True, # because m3u8
+        },
+    }]
+
+    def _real_extract(self, url):
+        playlist_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, playlist_id)
+
+        title = self._og_search_property('title', webpage)
+        description = self._og_search_property('description', webpage)
+
+        javascript = self._download_webpage(
+            'http://www.stolaf.edu/multimedia/components/eventlib.cfc'
+            '?method=getPlayerPlaylist&eventtype=e&eventid=%s&html5stream=%u' % (playlist_id, 3),
+            playlist_id, 'Downloading playlist #%u' % (3))
+        thePlaylist = self._parse_json(
+            self._search_regex(r'(?s)thePlaylist\s*=\s*(\[.*?\]);', javascript, 'thePlaylist'),
+            playlist_id, transform_source=js_to_json)
+        # token = self._search_regex(r'n7kIjJed73\s*=\s*\'(.*?)\';', javascript, 'token')
```

Comment:
```js
	function streamPlayer6jw () {
		jwplayer('player').setup({
			/* ... */
			rtmp: {
				securetoken: n7kIjJed73
			},
			/* ... */
		});
	}
```

It needs to be passed to the `--token` option of `rtmpdump` for RTMP downloading to work.